c++/src/H5Attribute.cpp

```diff
@@ -11,6 +11,7 @@
  * help@hdfgroup.org.                                                        *
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
+#include <cstdlib>
```

Comment:
Needed for free(). Not having this doesn't break the build, but only due to accidental inclusion via other files.